wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/EventLogScreen.kt

```diff
@@ -1,95 +1,233 @@
+
+    // Manually format the LocalDateTime to the desired output format
+    val dayOfWeek = localDateTime.dayOfWeek.name.take(3).lowercase().replaceFirstChar { it.uppercase() } // e.g., "Wed"
+    val month = localDateTime.month.name.take(3).lowercase().replaceFirstChar { it.uppercase() }         // e.g., "Oct"
+    val day = localDateTime.dayOfMonth                                // e.g., "02"
+    val hour = if (localDateTime.hour > 12) localDateTime.hour - 12 else localDateTime.hour
+    val minute = localDateTime.minute.toString().padStart(2, '0')     // e.g., "20"
+    val amPm = if (localDateTime.hour >= 12) "PM" else "AM"
+
+
+    return "$dayOfWeek $month $day $hour:$minute $amPm"
+}
+
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun EventLogScreen(documentId: String, documentModel: DocumentModel, onNavigate: (String) -> Unit) {
+fun EventLogScreen(
+    documentId: String,
+    documentModel: DocumentModel,
+    navController: NavHostController
+) {
     val eventInfos = remember {
         mutableStateListOf<EventInfo>().apply {
             addAll(documentModel.getEventInfos(documentId))
         }
     }
 
-    Box(
-        modifier = Modifier.fillMaxHeight()
+    ScreenWithAppBarAndBackButton(
+        title = stringResource(R.string.document_info_screen_menu_activities_log),
+        onBackButtonClick = { navController.popBackStack() }
     ) {
-        ScreenWithAppBarAndBackButton(
-            title = stringResource(R.string.document_info_screen_menu_activities_log),
-            onBackButtonClick = { onNavigate(WalletDestination.PopBackStack.route) }
-        ) {
-            if (eventInfos.isEmpty()) {
-                Row(
-                    modifier = Modifier.fillMaxWidth(),
-                    horizontalArrangement = Arrangement.Center,
-                ) {
-                    Text(
-                        modifier = Modifier.padding(8.dp),
-                        text = stringResource(R.string.document_info_screen_menu_activities_log_screen_empty),
-                        style = MaterialTheme.typography.titleLarge,
-                        textAlign = TextAlign.Center
-                    )
-                }
-            } else {
-                Column(modifier = Modifier.weight(1f)) {
+        if (eventInfos.isEmpty()) {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.Center,
+            ) {
+                Text(
+                    modifier = Modifier.padding(8.dp),
+                    text = stringResource(R.string.document_info_screen_menu_activities_log_screen_empty),
+                    style = MaterialTheme.typography.titleLarge,
+                    textAlign = TextAlign.Center
+                )
+            }
+        } else {
+            Column(
+                verticalArrangement = Arrangement.Top,  // Aligns content to the top
```

Comment:
This comment is just restating the obvious.
See go/tott/563.

Reply:
Done.

Reply:
Adding screenshots of the UI.
![Details](https://github.com/user-attachments/assets/2538e501-9f91-4da5-b10e-1383ce7086dd)
![Show_Usage](https://github.com/user-attachments/assets/02bb774b-5af2-4961-9826-5a81559cee86)